cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactVersion.java

```diff
@@ -30,11 +30,12 @@
  * parse version string in format of [major].[minor].[fix](-|.)[suffix]
  * }
  * </pre>
+ * Note that the suffix can not have numerical values in it.
  */
 @Beta
 public final class ArtifactVersion implements Comparable<ArtifactVersion> {
```

Comment:
Seems like its just a renaming, with PluginVersion left in place (but deprecated) for backwards compat.